TheAurorian/src/main/resources/assets/theaurorian/patchouli_books/the_aurorian_guide/zh_cn/entries/modprogression/7moontemplemobs.json

```
@@ -0,0 +1,27 @@
+{
+    "name": "月宫中的生物",
+    "icon": "theaurorian:bossspawnerkeeper",
+    "category": "2modprogression",
+    "sortnum": "7",
+    "readbydefault": true,
+    "pages": [
+        {
+            "type": "entity",
+            "entity": "theaurorian:moonacolyte",
+            "text": "$(#008cff)皎月侍从$()会击退你并在攻击到你时让你漂浮起来。"
+        },
+        {
+            "type": "entity",
+            "entity": "theaurorian:crystallinesprite",
+            "text": "$(#008cff)晶魂$()会在空中向玩家发射火焰。"
+        },
+        {
+            "type": "text",
+            "text": "$(#8800ff)皎月女王$()是$(#008cff)月宫$()的 BOSS。她会使用自己的护盾将你围起来并 perform shield charges 以将你击飞至空中！"
```

|Feedback Priority|
|---|
|⛰ Mountain / **Blocking and requires immediate action**|

```suggestion
            "text": "$(#8800ff)皎月女王$()是$(#008cff)月宫$()的 BOSS。她会使用自己的护盾将你围起来并充能以将你击飞至空中！"
```
删掉的这一句我不会，需要帮助。

---

我感觉意思是对着敌人充能。

---

就按着你的意思翻

---

OK